src/controllers/DefaultController.php

```diff
@@ -8,6 +8,7 @@ public function handle(Request $request, $db)
 
         // just add the available methods, with links
         $retval['events']          = $request->base . '/' . $request->version . '/events';
+        $retval['categories']      = $request->base . '/' . $request->version . '/categories';
```

Comment:
I'm not sure how useful it is to offer this link at the entry level of the API, I mostly think of this as a list of places a user wants to go